from_cpython/Include/Python.h

```diff
@@ -76,6 +76,7 @@
 #include "descrobject.h"
 #include "warnings.h"
 #include "weakrefobject.h"
+#include "socketmodule.h"
```

Comment:
This should probably get included in the files that need it rather than in Python.h, since it's not part of CPython's Python.h.